geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -1592,6 +1596,15 @@ public Object selectValue(String queryPredicate) throws FunctionDomainException,
    */
   public void registerInterest(K key, InterestResultPolicy policy);
 
+
+  public default void registerInterestForAllKeys() {
+    registerInterestRegex(".*");
+  }
+
+  public default void registerInterestForKeys(Iterable<K> iterable) {
+    iterable.forEach(k -> registerInterest(k));
+  }
+
```

Comment:
Do we need API like:

registerInterestForAllKeys(InterestResultPolicy policy)
registerInterestForKeys(Iterable<K> iterable, InterestResultPolicy policy)